README.md

```diff
@@ -110,18 +110,19 @@ Phone numbers can be deserialized from an international format by setting the ty
 
 ### Using `libphonenumber\PhoneNumber` objects in forms
 
-You can use the `tel` form type to create phone number fields. There are two widgets available.
+You can use the `PhoneNumberType` (`tel` for Symfony 2) form type to create phone number fields. There are two widgets available.
```

Comment:
The class name should be used in Symfony 2.8 too (you could still use the alias but it will trigger deprecations).